data/102/063/431/102063431.geojson

```diff
@@ -10,6 +10,12 @@
     "geom:latitude":49.341875,
     "geom:longitude":8.152294,
     "iso:country":"DE",
+    "label:deu_x_preferred_longname":[
+        "Kreisfreie Stadt Neustadt an der Weinstra\u00c3e"
```

Comment:
This seems awkward to repeat `Stadt` and `Neustadt`?

See also:

- https://de.wikipedia.org/wiki/Landkreis_Neustadt_an_der_Weinstra%C3%9Fe

Reply:
Yes, drop the extra `Stadt`

Reply:
Similar to the double "kreis", I'll check for "stadt" in the `wof:name` and QA the label values.